src/index.html

```diff
@@ -2,45 +2,107 @@
+                    <option value="None" selected="yes">None</option>
+                    <option value="HorizontalVertical">Horizontal-Vertical</option>
+                    <option value="Horizontal">Horizontal</option>
+                    <option value="Vertical">Vertical</option>
+                    <option value="Center">Center</option>
+                  </select>
+                </label>
+              </a>
+            </li>
+            <li class="mr-3 flex-1">
+              <a href="#" class="block py-1 md:py-3 pl-0 md:pl-1 align-middle text-white no-underline border-b-2 border-gray-800 border-red-500">
+                <label for="fit-margin">Fit margin:
+                  <input type="number" id="fit-margin" min="0" max="100" value="0" maxlength="3" oninput="validity.valid||(value='');" class="text-red-900 pl-1 float-right md:w-1/3">
+                </label>
+              </a>
+            </li>
+            <li class="mr-3 flex-1">
+              <div id="zoom-config-controls" class="py-1 md:py-3 pl-1 md:pl-1 align-middle text-white border-b-2 border-gray-800 border-yellow-500">
+                <label for="zoom-throttle" class="block w-full">Throttle:
+                  <input id="zoom-throttle" class="md:w-1/3 pl-1 float-right bg-gray-500 text-black" type="number" placeholder="throttle" value="30" min="0" max="100" disabled
+                         title="to play with throttle pass parameter in url like this: .../?zoomThrottle=40"/>
+                </label>
+                <label for="zoom-debounce" class="block w-full mt-3">Debounce:
+                  <input id="zoom-debounce" class="md:w-1/3 pl-1 float-right bg-gray-500 text-black" type="number" placeholder="debounce" value="30" min="0" max="100" disabled
+                         title="to play with debounce pass parameter in url like this: .../?zoomDebounce=30"/>
+                </label>
+              </div>
+            </li>
+          </ul>
         </div>
-        <p class="info">(either drop a file here)</p>
       </div>
-    </div>
 
-    <div style="position:relative; width:100%; height:800px">
-      <div id="bpmn-container"></div>
-    </div>
+      <!-- Main section -->
+      <div class="main-content flex-1 bg-gray-100 mt-12 md:mt-2 pb-24 md:pb-5">
+        <div class="bg-gray-800 pt-3">
+          <div class="rounded-tl-3xl bg-gradient-to-r from-purple-900 to-gray-800 p-4 shadow text-2xl text-white">
+            <h3 class="font-bold pl-2">Diagram</h3>
+          </div>
+        </div>
 
+        <div class="flex flex-row flex-wrap flex-grow mt-2">
+          <div class="w-full p-6">
+            <!--Graph Space-->
+            <!-- TODO: check if possible to drop [style]-->
+            <div class="bg-white border-transparent rounded-lg shadow-xl overflow-hidden absolute" style="top: 158px; right: 20px; bottom: 20px; left: 215px;">
+              <div class="p-5">
```

Comment:
:warning: this padding is nice but it seems to hide the fit margin feature.
Could we remove it and set a non zero fit margin at startup? that way we will have the same visual effect but we would also be able to see the impact of a zero fit margin
@aibcmars what do you think?

Reply:
ah yes good idea:) 👍